README.md

```diff
@@ -82,10 +82,10 @@ library for it project:
 * **WITH_PROXY** (default OFF): Include proxy support in the library.
 * **WITH APPS** (default OFF): Build with sample applications.
 * **WITH_PROXY_APPS** (default OFF):Build with proxy sample applications.
-* **WITH_VIRTIO_MASTER** (default ON): Build with virtio master enabled.
+* **WITH_VIRTIO_DRIVER** (default ON): Build with virtio driver (front-end)  enabled.
   This option can be set to OFF if the only the remote mode is implemented.
-* **WITH_VIRTIO_SLAVE** (default ON): Build with virtio slave enabled.
-  This option can be set to OFF if the only the master mode is implemented.
+* **WITH_VIRTIO_DEVICE** (default ON): Build with virtio device (back-end)  enabled.
+  This option can be set to OFF if the only the driver mode is implemented.
```

Comment:
`Front-end` and `back-end` terms can be ambigious.
suggestion:
 Build with virtio driver (virtio host role) enabled. 
 Build with virtio device (virtio guest role) enabled.

Reply:
> `Front-end` and `back-end` terms can be ambigious.
> suggestion:
> Build with virtio driver (virtio host role) enabled.
> Build with virtio device (virtio guest role) enabled.

This is the same wording as in the original. I'd like to keep this commit focused on the variable name changes.

Reply:
> > `Front-end` and `back-end` terms can be ambigious.
> > suggestion:
> > Build with virtio driver (virtio host role) enabled.
> > Build with virtio device (virtio guest role) enabled.
> 
> This is the same wording as in the original. I'd like to keep this commit focused on the variable name changes.
In this case could you just suppress the Front-end` and `back-end` terms?